src/main/java/org/antlr/intellij/plugin/preview/InputPanel.java

```diff
@@ -266,17 +261,7 @@ public void switchToGrammar(PreviewState previewState, VirtualFile grammarFile)
 	}
 
 	public void setEditorComponent(JComponent editor) {
-		BorderLayout layout = (BorderLayout) outerMostPanel.getLayout();
-		String EDITOR_SPOT_COMPONENT = BorderLayout.CENTER;
-		// atomically remove old
-		synchronized (swapEditorComponentLock) {
-			Component editorSpotComp = layout.getLayoutComponent(EDITOR_SPOT_COMPONENT);
-			if ( editorSpotComp!=null ) {
-				editorSpotComp.setVisible(false);
-				outerMostPanel.remove(editorSpotComp); // remove old editor if it's there
-			}
-			outerMostPanel.add(editor, EDITOR_SPOT_COMPONENT);
-		}
+		editorSplitter.setFirstComponent(editor);
```

Comment:
I love the simplification of this code!

Reply:
Right, that was a pleasant surprise!

Reply:
I just noticed a problem though: the bottom console grows automatically with each new line of text that is added inside. That's not really desired

Reply:
Is it possible to set a maximum height?